test/python/algorithms/test_vqe.py

```diff
@@ -277,6 +271,27 @@ def test_with_aer_qasm_snapshot_mode(self):
         result = vqe.compute_minimum_eigenvalue(operator=self.h2_op)
         self.assertAlmostEqual(result.eigenvalue.real, self.h2_energy, places=6)
 
+    @data(L_BFGS_B(), P_BFGS())
```

Comment:
Is this another slow test? I think we could test this more efficiently. Is it not enough to do a single step with the gradients passed? If one step succeeds, we fixed the problem, since before there was a type error.

Reply:
Thanks. right. this test is too slow. I changed it to `maxiter/maxfun=1` and ignored the optimized value.